assessment.js

```diff
@@ -0,0 +1,112 @@
+document.addEventListener("DOMContentLoaded", () => {
+    console.log("DOMContent is loaded")
+    selectTitle()
+    // displaySelectedFilm(film)                                                                                                                                                                                      select", selectTitle)
+    // let reviewInput = document.querySelector("#text-input")
+    // let form = document.querySelector("form")
+    // form.addEventListener("submit", (event) => {
+    //       let textInput = document.querySelector("#text-input")
+    //textInput.innerText = ""
+    let inputBtn = document.querySelector("#btn-input")
+    inputBtn.addEventListener("submit", (reviewSection))
+    //event.preventDefault()
+    //   if (textInput.innerText = "") {
+    //      form.appendChild(textInput)
+    // })
+
+    // console.log("review box", reviewInput)
+    // reviewInput.addEventListener("click",  {
+
+    // })
+})
+
+const selectTitle = () => {
+    let url = "https://ghibliapi.herokuapp.com/films"
+    fetch(url)
+        .then(response => {
+            console.log(response)
+            return response.json()
+        })
+        .then(film  => {
+            console.log(film)
+            displaySelectedFilm(film)
+        })
+        .catch(error => {
+            console.log("there is an error", error)
+        })
+    //console.log("is it the right url?", url)
+}
+
+const displaySelectedFilm = (film) => {
+    let selectBox = document.querySelector(".title-box")
+    let firstDiv = document.querySelector(".select")
+    //let newSelectBox = document.createElement("select")
+    let infoSection = document.querySelector(".info-section")
+    let newInfoSection = document.createElement("div")
+
+    for (let i = 0; i < film.length; i++) {
+        let filmOptions = document.createElement("option")
+        filmOptions.innerText = (film[i].title)
+        selectBox.appendChild(filmOptions)
+        //    let filmTitle = document.createElement("h3")
+        //    filmTitle.innerText = film[i].title
+        //    let releaseYr = document.createElement('p')
+        //    releaseYr.innerText = film[i].release_date
+        //    let filmDes = document.createElement("p")
+        //    filmDes.classList.add('description')
+        //    filmDes.innerText = film[i].description
```

Comment:
Large chunks of commented code makes is hard for a reviewer to navigate your code. We should never leave commented code at the end our task. Comments should be used for leaving quick clarifying notes accompanying our code.